README.md

```diff
@@ -1941,6 +1941,51 @@ Usage typically looks like:
     }
 ~~~
 
+##### `kerb`
+
+Enable mod_auth_kerb parameters for a virtual host. Valid values are 'true' or 'false'. Defaults to 'false'.
+
+Usage typically looks like:
+
+~~~ puppet
+    apache::vhost {'sample.example.net':
+       kerb => true,
+       krb_method_negotiate => 'on',
+       krb_auth_realms => ['EXAMPLE.ORG'],
+       krb_local_user_mapping         => 'on',
+       directories  => { path         => '/var/www/html',
+                         auth_name    => 'Kerberos Login',
+                         auth_type    => 'Kerberos',
+                         auth_require => 'valid-user',
+
```

Comment:
Can you fix the arrow alignment here? There should be a newline after {, only one level of indentation for the hash key/values, and the hash arrows should be aligned independently from the resource attribute arrows.